edit_questionpy_form.php

```diff
@@ -129,6 +129,10 @@ protected function definition_inner($mform): void {
         $mform->addElement('hidden', 'qpy_package_hash', '');
         $mform->setType('qpy_package_hash', PARAM_RAW);
 
+        // Whether the content of a qpy:shuffle-contents can be randomly shuffled.
+        $mform->addElement('hidden', 'shuffleanswers', get_config('qtype_questionpy', 'shuffleanswers'));
+        $mform->setType('shuffleanswers', PARAM_BOOL);
+
```

Comment:
Bitte keine neuen Einstellungen einführen. Ob so eine Einstellung für das jeweilige Paket sinnvoll ist, müssen die Pakete selber wissen und in ihren Options definieren.